src/ontology/mondo-edit.obo

```diff
@@ -279835,19 +279830,27 @@ property_value: confidence "0.09890109890109877" xsd:double
 id: MONDO:0014243
 name: Prader-Willi syndrome due to point mutation
 subset: ordo_etiological_subtype {source="Orphanet:398069"}
+synonym: "arthrogryposis, distal, with hypopituitarism, intellectual disability, and facial anomalies" EXACT [MONDO:0008820, OMIM:208080]
+synonym: "arthrogryposis, distal, with hypopituitarism, mental retardation, and facial anomalies" EXACT DEPRECATED [OMIM:208080]
+synonym: "Chitayat-Hall syndrome" RELATED [GARD:0010087]
+synonym: "distal arthrogryposis with hypopituitarism, intellectual disability and facial anomalies" RELATED [GARD:0010087]
 synonym: "Prader-Willi syndrome due to a point mutation" RELATED [Orphanet:398069]
```

Comment:
this can be exact

Reply:
I actually deleted "Prader-Willi syndrome due to a point mutation" synonym because the name was the same as the label name.